zkp-circuits/src/metadata.rs

```diff
@@ -7,6 +11,7 @@ use nimiq_serde::{Deserialize, Serialize};
 /// This data structure holds metadata about the verifying keys.
 /// It can be used to check whether verifying keys are still up to date.
 #[derive(Serialize, Deserialize, PartialEq, Eq, Clone, Debug)]
+#[serde(rename_all = "camelCase")]
```

Comment:
If this is only used for `json` then I guess `vks_commitment` can be a string a `blocks_per_epoch` doesn't require `#[serde(with = "nimiq_serde::fixint::be")]`